### ISSUE TITLE
bug with no-duplicate-selectors

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#931]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Throw error if use attribute selector.

> Which rule, if any, is this issue related to?

`no-duplicate-selectors`

> What CSS is needed to reproduce this issue?

```scss
[disabled].foo,
[disabled] .foo {
    background: #b8b8b8;
}

```
Also reproduced in pure CSS.

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "no-duplicate-selectors": true
  }
}
```

> Which version of stylelint are you using?

`5.0.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS plugin

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SCSS

> What did you expect to happen?

No errors.

> What actually happened (e.g. what warnings or errors you are getting)?

Throw error.

```shell
_test.scss
1:1	Unexpected duplicate selector "[disabled] .foo" (no-duplicate-selectors)
```

**Comment by smutnyleszek** [comment]
The same goes with:

```css
ul,
ol {
    margin-left: 1rem;
    padding: 0;
}

ul {
    list-style-type: disc;
}
```

I would expect `ul, ol` to be identified as completely different selector than `ul`.

**Comment by davidtheclark** [comment]
@evilebottnawi or @smutnyleszek : If you can file a PR with at least failing tests that illustrate the need, that would really help speed up this fix.

**Comment by davidtheclark** [comment]
@evilebottnawi I've reproduced the bug in a test. Will try to fix before breakfast.

@smutnyleszek Your case (`ul, ol {} ul {}`) passes. If you are still having the problem, please file another information with more information and ideally a failing test case.

**Comment by davidtheclark** [comment]
The problem ended up being not with attribute selectors but with the descendant combinator. I was normalize the selectors by brutally removing spaces, but those particular spaces are important. So the more accurate https://github.com/getify/normalize-selector function did a better job.
